app/services/katello/pulp3/content_view_version/metadata_generator.rb

```diff
@@ -43,9 +43,10 @@ def generate!
 
         def generate_repository_metadata(repo)
           repo.slice(:name, :label, :description, :arch, :content_type, :unprotected,
-                     :checksum_type, :os_versions, :major, :minor).
+                     :checksum_type, :os_versions, :major, :minor, :product_id, :content_id).
```

Comment:
```suggestion
                     :checksum_type, :os_versions, :major, :minor).
```

Don't need these since you are capturing it under `content`